scripts/launch/node.sh

```diff
@@ -112,7 +112,7 @@ educator_params="
 --student-api-no-auth 3BAyX5pNpoFrLJcP5bZ2kXihBfmBVLprSyP1RhcPPddm6Dw42jzEPXZz22
 --publication-period 15s
 --pdf-resource-path ../pdfs/template
---cert-issuer-name "Grimpy Cat University"
+--cert-issuer-name \"University\"
 --cert-issuer-url example@gmail.com
```

Comment:
I forgot to comment on this, but AFAIK it is good practice to use `*@example.com` for these cases.
Also I took both this values from `certificateIssuerInfoEx` (in `Dscp.Core.Arbitrary`) so you may want to change those too.

Reply:
Oh ok, didn't know about that practice. Suggest some e-mail/website then :)

I had to set `University` here because I don't know how to escape a quoted expression containing spaces in bash (i.e. `... \"Grimpy Cat University\"` failed to interpret).

Reply:
I don't know, `university@example.com`?
You are right about the escaping, I totally forgot about that.
On the same matter, should't `--sql-conn-str "postgresql:///$sql_db_name"`, a few lines above, be escaped as well?

Reply:
Yeah, I like it.

Nice point, I somehow forgot about escaping there even despite emacs highlights it differently. Truly surprised it worked :smile:

Reply:
:man_shrugging: